src/main/java/com/naveenautomation/Pages/MyAccountPage.java

```diff
@@ -12,15 +12,15 @@ public class MyAccountPage extends Page {
 	public MyAccountPage(WebDriver wd, boolean waitForPageToLoad) {
 		super(wd, waitForPageToLoad);
 	}
-
 	
 	private static final By phonelink = By.cssSelector("i.fa.fa-phone");
 	private static final By GiftCertificatesLink = By
 			.cssSelector("footer div.row div.col-sm-3:nth-of-type(3) ul li:nth-of-type(2) a");
 	private static final By myAccountText = By.xpath("//h2[text()='My Account']");
 	private static final By changePasswordLink = By.cssSelector("#column-right a:nth-of-type(3)");
-	private static final By passwordChangesSuccessBanner = By.cssSelector("div.alert-success");
+	private static final By SuccessBanner = By.cssSelector("div.alert-success");
```

Comment:
naming convention is not correct and why you want to change the name

Reply:
Based on the review I updated the code. That locator is common for all the banner that is why I changed the name